lighthouse-core/audits/content-width.js

```diff
@@ -38,17 +38,9 @@ class ContentWidth extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (typeof artifacts.ContentWidth === 'undefined' ||
-        typeof artifacts.ContentWidth.scrollWidth === 'undefined' ||
```

Comment:
do you want to limit ourselves to just removing for now since that's the only check that's been added? 
stripping out other error checks comes after standardizing gatherer output next :)